exercises/raindrops/canonical-data.json

```diff
@@ -1,74 +1,92 @@
 {
    "cases": [
       {
+         "description" : "the sound for 1 is 1",
          "number" : 1,
          "expected": "1"
       },
       {
+         "description" : "the sound for 3 is Pling",
          "number" : 3,
          "expected": "Pling"
       },
       {
+         "description" : "the sound for 5 is Plang",
          "number" : 5,
          "expected": "Plang"
       },
       {
+         "description" : "the sound for 7 is Plong",
          "number" : 7,
          "expected": "Plong"
       },
       {
+         "description" : "the sound for 6 is Pling as it has a factor 3",
          "number" : 6,
          "expected": "Pling"
       },
       {
+        "description" : "the sound for 8 is 8",
         "number" : 8,
         "expected": "8"
       },
       {
+         "description" : "the sound for 9 is Pling as it has a factor 3",
          "number" : 9,
          "expected": "Pling"
       },
       {
+         "description" : "the sound for 10 is Plang as it has a factor 5",
          "number" : 10,
          "expected": "Plang"
       },
       {
+         "description" : "the sound for 14 is Plong",
```

Comment:
I rather like the cases for 9, and 10, where there is an explanation for the sound in the description. It is missing here, is this by design?